proto/cosmos/orm/module/v1alpha1/module.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package cosmos.orm.module.v1alpha1;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module defines the ORM module which adds providers to the app container for
+// module-scoped DB's. In the future it may provide gRPC services for interacting
+// with ORM data.
+message Module {
+  option (cosmos.app.v1alpha1.is_module) = {
```

Comment:
coming back to this, im wondering if is_module is the best name for the option. we use `orm.whatever.table` and `orm.whatever.singleton`, and not `orm.whatever.is_table`, which seems like an odd difference 

wdyt?

Reply:
Yeah, is_module is weird. Okay so just `module`?

Reply:
yeah, just `module` is what i was thinking

Reply:
updated